src/test/java/core/basesyntax/service/FileWriterServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceTest {
+
+    private static final String PATH_TO_WRITE = "src/test/resources/FileWriter.csv";
+    private FileWriterService fileWriter;
+
+    @BeforeEach
+    void setUp() {
+        fileWriter = new FileWriterServiceImpl();
+    }
+
+    @Test
+    void write_checkCorrectWriting_ok() {
+        String textForTest = "r,banana,20";
+        fileWriter.write(textForTest, PATH_TO_WRITE);
+        List<String> exceptedData = List.of("r,banana,20");
+        List<String> actualData;
+        try {
+            actualData = Files.readAllLines(new File(PATH_TO_WRITE).toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file! ",e);

```

Comment:
After running tests you need to clear file where you wrote data, you don`t need this file more